exoscale/client.go

```diff
@@ -62,3 +64,11 @@ func GetDNSClient(meta interface{}) *egoscale.Client {
 	}
 	return config.dnsClient
 }
+
+func environment(meta interface{}) string {
```

Comment:
To stay consistent with the other `getXXXX` helpers:

```suggestion
func getEnvironment(meta interface{}) string {
```